doc/book/connectors/__python.rst

```diff
@@ -44,3 +44,214 @@ Also there are several community-driven Python connectors:
 * `asynctnt <https://github.com/igorcoding/asynctnt>`__ with asyncio support
 * `aiotarantool <https://github.com/shveenkov/aiotarantool>`__ also with asyncio support
 * `gtarantool <https://github.com/shveenkov/gtarantool>`__ with gevent support **no active maintenance**
+
+Feature comparison
+------------------
+
+Last update: December 2021
+
+..  list-table::
+    :header-rows: 1
```

Comment:
You can also add `:stub-columns: 1` to change style of the left column

Reply:
Added